charts/external-dns/values.yaml

```diff
@@ -0,0 +1,50 @@
+external-dns:
+  ## AWS configuration to be set via arguments/env. variables
+  ##
+  aws:
```

Comment:
@LutzLange how do expect the authz part work for demo environments?

I would imagine there is a role to be assumed?

Reply:
This would be my assumption as well. I'm not specialist for AWS IAM. But we could look for a matching role together.

Reply:
We are looking at implementing IRSA for this. So we would need to create an IAM role with the required permission in our AWS account and attach the correct annotiations when deploying external-dns.